lib/web_ui/lib/src/engine/raw_keyboard.dart

```diff
@@ -88,6 +88,14 @@ class RawKeyboard {
     return _onMacOs;
   }
 
+  // During IME composition, Tab fires twice (once for composition and once
+  // for regular tabbing behavior), which causes issues. Intercepting the
+  // tab keydown event during composition prevents these issues from occurring.
+  // https://developer.mozilla.org/en-US/docs/Web/API/Element/keydown_event#ignoring_keydown_during_ime_composition
```

Comment:
This comment describes an implementation detail of the function. I think it should go inside the function. What do you think?

Reply:
That definitely makes sense, I was just following the pattern of the `_shouldDoKeyGuard()` function where the comment is included right above the function.  I don't have a strong preference either way